### PR TITLE
Change top toolbar UI to match Figma

### PR DESCRIPTION
**Pull request by amoliksingh**
aframe inspector:
image
figma:
image
Loom video:
https://www.loom.com/share/82005488ad17432fb53386b84c8100e5

### CODE REVIEW

**Comment by amoliksingh** [comment]
Should I move the save button to the top in this same PR?

**Review by drbalar — DISMISSED** [review]
Looks good 🎨

File: src/components/scenegraph/Toolbar.js [hunk 1]
```diff
@@ -397,7 +397,7 @@ export default class Toolbar extends React.Component {
     return (
       <div id="toolbar">
         <div className="toolbarActions">
-          <a
+          {/* <a
```
Comment (ahmedhamodi) [inline src/components/scenegraph/Toolbar.js]: Should we remove this?

File: src/components/viewport/TransformToolbar.js [hunk 2]
```diff
@@ -8,15 +10,318 @@ var TransformButtons = [
   { value: 'rotate', icon: 'fa-repeat' }
 ];
 
+function getCookie(name) {
```
Comment (ahmedhamodi) [inline src/components/viewport/TransformToolbar.js]: As mentioned off line, if this is copied logic from elsewhere, it should be removed from that location. If we can export from other file and import here, that would be more ideal so we don't need to review every line here again

**Review by ahmedhamodi — COMMENTED** [review]
UI code changes look good, but can't approve until the functional changes are tested/agreed upon. See offline discussion

File: src/components/scenegraph/Toolbar.js (hunk 1 above)
Comment (amoliksingh) [reply src/components/scenegraph/Toolbar.js]: removed

File: src/components/scenegraph/Toolbar.js [hunk 3]
```diff
@@ -233,146 +173,6 @@ export default class Toolbar extends React.Component {
     Events.emit('entitycreate', { element: 'a-entity', components: {} });
   }
```
Comment (ahmedhamodi) [inline src/components/scenegraph/Toolbar.js]: You can remove addEntity function now

Comment (amoliksingh) [reply src/components/scenegraph/Toolbar.js]: removed

**Review by ahmedhamodi — APPROVED** [review]

**Pull request merged**